components/support/rc_crypto/nss/src/ec.rs

```diff
@@ -96,6 +101,7 @@ pub fn generate_keypair(curve: Curve) -> Result<(PrivateKey, PublicKey)> {
     // https://searchfox.org/mozilla-central/rev/f46e2bf881d522a440b30cbf5cf8d76fc212eaf4/dom/crypto/WebCryptoTask.cpp#2389
     let mech = match curve {
```

Comment:
Let's remove that `match` statement and inline `CKM_EC_KEY_PAIR_GEN` directly, I think it was a mistake in the first place.

Reply:
done